### PR TITLE
[DOC] Refine documentation of runtime feature detection

### PR DESCRIPTION
**Pull request by larroy**
## Description ##

This refines the documentation of the Python API.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by larroy** [comment]
@aaronmarkham

**Comment by roywei** [comment]
@mxnet-label-bot add[Doc, pr-awaiting-review]

File: python/mxnet/runtime.py [hunk 1]
```diff
@@ -27,17 +27,27 @@
...
     _fields_ = [
         ("_name", ctypes.c_char_p),
-        ("enabled", ctypes.c_bool)
+        ("_enabled", ctypes.c_bool)
     ]
 
     @property
     def name(self):
+        """
+        feature name
```
Comment (aaronmarkham) [inline python/mxnet/runtime.py]:
```suggestion
        Feature name.
```
Can the name be anything? Spaces, special characters? Might be a good spot to provide more direction.

File: python/mxnet/runtime.py [hunk 2]
```diff
@@ -27,17 +27,27 @@
...
+        """
         return self._name.decode()
 
+    @property
+    def enabled(self):
+        """
+        True if MXNet was compiled with the given compile-time feature
```
Comment (aaronmarkham) [inline python/mxnet/runtime.py]:
```suggestion
        True if MXNet was compiled with the given compile-time feature.
```
Also, there's no example provided for this function.

File: python/mxnet/runtime.py [hunk 3]
```diff
@@ -50,7 +60,8 @@ def feature_list():
 
     Returns
     -------
-    :return: list of class LibFeature indicating which features are available and enabled
+    list
```
Comment (aaronmarkham) [inline python/mxnet/runtime.py]: Can you provide example usage?

**Comment by aaronmarkham** [comment]
One other thing - I couldn't find how to navigate to this API. I had to search for it.
It would make sense to have it listed here: 
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-14238/2/api/python/index.html#tools

**Comment by larroy** [comment]
@aaronmarkham There's a link in the page you showed: "Run-Time Feature detection / Library Info
Run-Time Feature detection / Library info
"

File: python/mxnet/runtime.py (hunk 3 above)
Comment (larroy) [reply python/mxnet/runtime.py]: Done

**Comment by vandanavk** [comment]
@aaronmarkham is this PR good to go?

**Pull request merged**